### PR TITLE
Add sequence diagram to README

### PR DESCRIPTION
**Pull request by rollulus**
Hi, I created a little diagram. Hope this clarifies things, and hope that my understanding is correct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,11 +54,19 @@ file\_sd\_configs can read and then then relabel as needed.
 
 ## How It Works
 
-The client registers with the proxy, and awaits instructions.
-
-When Prometheus performs a scrape via the proxy, the proxy finds
-the relevant client and tells it what to scrape. The client performs the scrape,
-sends it back to the proxy which passes it back to Prometheus.
+![Sequence diagram](./docs/sequence.svg)
```
Comment (brian-brazil) [inline README.md]: I'd say Network rather than NAT, as it's more general.

Comment (rollulus) [reply README.md]: Fixed :+1:

Comment (rollulus) [reply README.md]: Apologies, let me re-create that svg, I damaged something else, one moment

**Comment by rollulus** [comment]
Alright, "NAT" changed into "network" and fixed the overlapping texts that I accidentally introduced

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**